packages/react-components/react-field/src/components/Field/Field.test.tsx

```diff
@@ -1,8 +1,18 @@
 import * as React from 'react';
+
```

Comment:
Putting this here because it's a suggestion for a new test case.

In the PR comment you describe how having an intermediary control in `Field`, e.g.:

```tsx
<Field>
  <Intermediary>
     <InputControl/>
  </Intermediary>
</Field>
```

was a driver of the context approach. We should probably have a test case to exercise this scenario.

Reply:
Added a test that includes a div between the Field and its component, and another that includes a Tooltip.